x/distribution/types/keys.go

```diff
@@ -1,8 +1,13 @@
 package types
 
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/tendermint/tendermint/crypto"
+)
+
 const (
 	// ModuleName is the module name constant used in many places
-	ModuleName = "distr"
+	ModuleName = "distribution"
```

Comment:
breaking API change

Reply:
Why are we doing this? Why can't we just keep the module name as is?

Reply:
The module account takes uses the takes the hash module name its address. I changed it because changing it later would require manually transferring the funds to a new account.

I recall that originally this module was called `distr` but when we renamed it, we forgot to change this constant. This is the only module where the name of the directory differs with the `ModuleName`.